versioned/interface.ts

```diff
@@ -161,9 +161,9 @@ abstract class Versioned {
               }
               const p = new Parser();
               try {
-                const _ = p.parseScript(x);
+                p.parseScript(x);
               } catch {
-                return "Bad lambda";
+                return "Unable to parse the lambda";
```

Comment:
In which case, I'll see this error on UI? It doesn't seem to work on UI.

Reply:
You can try with this input `qdfqdf!`, it's working on my side. Also you may need to blur the input to trigger a check

Reply:
close the thread, as the new request is to remove lambda execution.